substrate/frame/system/src/lib.rs

```diff
@@ -1916,7 +1896,8 @@ impl<T: Config> Pallet<T> {
 	/// execution else it can have a large impact on the PoV size of a block.
```

Comment:
`event_no_consensus()` (above this comment) can be written to be much more efficient now, probably worth doing in case it impacts performance.